### PR TITLE
Document enabling wildcard routes in the web console

### PR DESCRIPTION
**Pull request by spadgett**
https://github.com/openshift/origin-web-console/pull/853
https://trello.com/c/GEdeKzea

### CODE REVIEW

File: install_config/router/default_haproxy_router.adoc [hunk 1]
```diff
@@ -846,6 +846,8 @@ $ oc env dc/router ROUTER_ALLOW_WILDCARD_ROUTES=true
 $ oc scale dc/router --replicas=1
 ----
 
+xref:../../install_config/web_console_customization.adoc#web-console-enable-wildcard-routes[Learn how to configure the web console for wildcard routes].
```
Comment (jwforres) [inline install_config/router/default_haproxy_router.adoc]: Is this consistent with how we normally link to other doc pages, seems like a lot of link text
cc @ahardin-rh ?

File: install_config/web_console_customization.adoc [hunk 2]
```diff
@@ -460,6 +460,29 @@ endif::[]
...
 endif::[]
 
+[[web-console-enable-wildcard-routes]]
+=== Enabling Wildcard Routes
+
+If you've enabled wildcard routes for a router, you can also enable wildcard
+routes in the web console. This lets users enter hostnames starting with an
+asterisk like `*.example.com` when creating a route. To enable wildcard routes:
+
+. Save this script to a file (for example, *_enable-wildcard-routes.js_*):
```
Comment (jwforres) [inline install_config/web_console_customization.adoc]: Looks like you are missing the actual script content

**Review by jwforres — CHANGES_REQUESTED** [review]

File: install_config/web_console_customization.adoc (hunk 2 above)
Comment (spadgett) [inline install_config/web_console_customization.adoc]: I see it?

Comment (jwforres) [inline install_config/web_console_customization.adoc]: nevermind, my eyes were skipping right over it

File: install_config/router/default_haproxy_router.adoc (hunk 1 above)
Comment (ahardin-rh) [inline install_config/router/default_haproxy_router.adoc]: @jwforres Yes, this seems verbose, but it is correct.

**Review by knobunc — APPROVED** [review]
LGTM.  Thanks!

**Review by jwforres — APPROVED** [review]

**Comment by jwforres** [comment]
@ahardin-rh ok technical review is done on this

**Comment by ahardin-rh** [comment]
Is this for 3.4?

**Comment by spadgett** [comment]
@ahaile-redhat it's for 3.5

**Comment by ahardin-rh** [comment]
Will do some minor follow-up edits in a follow-up PR. Thanks!

**Comment by ahardin-rh** [comment]
[rev_history]
|xref:../install_config/web_console_customization.adoc#install-config-web-console-customization[Customizing the Web Console]
|Added a new section on xref:../install_config/web_console_customization.adoc#web-console-enable-wildcard-routes[enabling wildcard routes] in the web console.
%

**Pull request merged**